rotkehlchen/tests/unit/decoders/test_zksync.py

```diff
@@ -104,3 +104,88 @@ def test_zksync_lite_deposit(database, ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_zksync_lite_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+    """Test a transaction with the Withdrawal event"""
+    tx_hex = deserialize_evm_tx_hash('0x234407968b9a688be3fb37cf7ff8ef3b4168d6cd85ec45b8344bb2a88832f982')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
```

Comment:
this looks like a duplication. We had an old format of doing this tests and we did weird things when upgrading them so maybe you copied it from one of those

```suggestion
    evmhash = deserialize_evm_tx_hash('0x234407968b9a688be3fb37cf7ff8ef3b4168d6cd85ec45b8344bb2a88832f982')  # noqa: E501
```